src/components/subscriptions/SubscriptionDetails.jsx

```diff
@@ -35,7 +35,7 @@ const SubscriptionDetails = ({ enterpriseSlug }) => {
           </Link>
         </Row>
       )}
-      <SubscriptionExpirationBanner />
+      <SubscriptionExpirationBanner subscriptionDetails />
```

Comment:
[for my own knowledge] I don't understand how this works or what value is actually in here.  I see that you made this have a boolean propType below, is it meant to pass in `hasMultipleSubscriptions`?

Reply:
@iloveagent57 When you include a prop like this without a value associated, it just tells React to default it to `true`. In `SubscriptionExpirationBanner`, this comes back as `props.subscriptionDetails = true`.

This is being used as the flag for whether the alerts are displayed from the contract or the plan detail page to differentiate the messaging as needed.

Reply:
Ah, neat!  Could we make the name have a verb in it so it's more obvious that it's a boolean?  Something like `messageIsDerviedFromSubscriptionPlan` (I know that's a mouthful)

Reply:
`isSubscriptionDetails` would be my suggestion, if anything. It's mostly common knowledge that a prop without a value is a boolean in the React community 🤷‍♂️